rules/framework.bzl

```diff
@@ -155,6 +155,12 @@ def _apple_framework_packaging_impl(ctx):
 
             # collect modulemaps
             for modulemap in dep[apple_common.Objc].direct_module_maps:
+                # rule_swift changed how non swift generates module map in this commit
```

Comment:
thanks for the comment here!

Reply:
I'm confused about the original implementation here, as well as the change. If we assign modulemap_in to the modulemap on every iteration, aren't we just assigning modulemap_in with the last value in the array `direct_module_maps` ?

Reply:
i have the same question actually! @segiddins